_posts/2013-05-29-release-notes-v2.11.0-M3.md

```diff
@@ -12,7 +12,7 @@ The Scala team and contributors [fixed 108 issues](https://issues.scala-lang.org
 
 Please give 2.11.0-M3 a spin! This release is *not* binary compatible with the 2.10.x series, so you will need to obtain builds of your dependencies. Once we start the release candidates, we will coordinate with the open source community to release these simultaneously, but for these milestones we are not asking library authors to go to that trouble.
 
-We'd love to hear about any regressions since 2.10.1. You can file bugs in [JIRA](https://issues.scala-lang.org/secure/CreateIssue.jspa?pid=10005&issuetype=1). Before doing so, please search for existing bugs and/or consult with the [scala-user](https://groups.google.com/forum/?fromgroups#!forum/scala-user) mailing list to be sure it is a geniune problem. Please set the 'Affects Version' field to 2.11.0-M3 and add the tag `regression`.
+We'd love to hear about any regressions since 2.10.1. You can file bugs in [Jira](https://issues.scala-lang.org/secure/CreateIssue.jspa?pid=10005&issuetype=1). Before doing so, please search for existing bugs and/or consult with the [scala-user](https://groups.google.com/forum/?fromgroups#!forum/scala-user) mailing list to be sure it is a genuine problem. Please set the 'Affects Version' field to 2.11.0-M3 and add the tag `regression`.
```

Comment:
Brand - Atlassian rebranded retroactively. (There's a similar desire by Apple to rebrand in favor of `macOS`, but I haven't suggested those changes because this repository didn't misspell the old Apple brands, unlike many other repositories...)

Reply:
Brand - Atlassian rebranded retroactively.